modules/tinymce/src/plugins/table/test/ts/browser/ResizeTableTest.ts

```diff
@@ -54,24 +56,32 @@ UnitTest.asynctest('browser.tinymce.plugins.table.ResizeTableTest', (success, fa
       lastObjectResizedEvent.set(e);
     };
 
+    const tableModified = (e: EditorEvent<{}>) => {
+      tableModifiedEvents.push(e);
+    };
+
     input.editor.on('ObjectResizeStart', objectResizeStart);
     input.editor.on('ObjectResized', objectResized);
+    input.editor.on('tablemodified', tableModified);
```

Comment:
```suggestion
    input.editor.on('TableModified', tableModified);
```
Lower case will work, but we should use the actual names in case we ever get around to adding plugin types. I probably missed this in previous PR's sorry.

Reply:
Ha, I almost commented on this and decided not to. It would be good to change it, yes.